docs/proposals/clusterid/README.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Adoption of ClusterID
+
+authors:
+- "@XiShanYongYe-Chang"
+- "@RainbowMango"
+
+reviewers:
+- TBD
+
+approvers:
+- TBD
+
+creation-date: 2022-06-16
+
+---
+
+# Adoption of ClusterID
+
+## Summary
+
+In the progress of registering a cluster into Karmada, we need to confirm the cluster identity to prevent the cluster from being added to Karmada repeatedly. [Kubernetes sig-multicluster](https://github.com/kubernetes/community/tree/master/sig-multicluster) have made a KEP: [ClusterId for ClusterSet identification](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid#kep-2149-clusterid-for-clusterset-identification) to identify a cluster in a Kubernetes-native way. We want to use this [ClusterID](https://github.com/kubernetes-sigs/about-api) to confirm the cluster identifier. 
+
+## Motivation
+
+Kubernetes' enhancements has proposed a [KEP-2149](https://github.com/kubernetes/enhancements/tree/master/keps/sig-multicluster/2149-clusterid). This KEP proposes an API called `ClusterProperty` which provides a way to store identification related, cluster scoped information. We can adopt this API to identify each cluster that joins Karmada.
+
+### Goals
+
+- Set the cluster identifier for the cluster that has been joined to Karmada.
+
+### Non-Goals
+
+- The API describing cluster properties upgrade.
+
+## Proposal
+
+### User Stories (Optional)
+
+#### Story 1
+
+In Karmada, a cluster can be joined many times with different name. For a cluster joined Karmada repeatedly, once we remove one of them, all the clusters will turn into unready. More detail info can refer to [issue](https://github.com/karmada-io/karmada/issues/880).
+
+So we need to verify the cluster uniques when cluster joining.
```

Comment:
The `User Story` is used to capture a description of a feature from a `user's perspective`. The user story describes the type of user, what they want and why. 
See [What's user story](https://www.techtarget.com/searchsoftwarequality/definition/user-story#:~:text=A%20user%20story%20is%20a,simplified%20description%20of%20a%20requirement.)

Reply:
As a cluster administrator, I want each cluster could be assigned with a unique ID, so that the registering process can be rejected if a cluster tries to register with another name.

For example.